features/marshal/marshalto.go

```diff
@@ -626,30 +626,38 @@ func (p *marshal) reverseListRange(expression ...string) string {
 }
 
 func (p *marshal) marshalBackward(varName string, varInt bool, message *protogen.Message) {
-	foreign := strings.HasPrefix(string(message.Desc.FullName()), "google.protobuf.")
+	local := p.IsLocalMessage(message)
 
 	p.P(`{`)
-	if foreign {
-		p.P(`encoded, err := `, p.Ident(generator.ProtoPkg, "Marshal"), `(`, varName, `)`)
-	} else {
+	if local {
```

Comment:
Is the surrounding pair of curly braces (line 631/662) necessary?
There is no variable declaration that could redeclare variables, or leak in the parent block. And it obfuscates the code a bit.

Reply:
Fixed :+1: